.github/workflows/ci.yml

```diff
@@ -0,0 +1,22 @@
+name: Continuous Integration 
+
+on:
+  pull_request:
+  schedule:
+    - cron: "0 0 * * *"
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.8'
+      - name: Install dependencies
+        run: pip install -e ".[dev,docs]"
```

Comment:
Is this step needed?

Reply:
I am using the dev packages here to install the specific version of tox that we are using.
Removing docs.